camel-sap/org.fusesource.camel.component.sap/src/org/fusesource/camel/component/sap/util/RfcUtil.java

```diff
@@ -1345,4 +1345,220 @@ public static String checkForKeywordLiterals(String value) {
 		}
 		return value;
 	}
+	
+	/**
+	 * Converts the JCo string representation for a default value into its
+	 * equivalent Java runtime value.
+	 * 
+	 * @param listMetaData
+	 *            - the meta-data for JCoList.
+	 * @param index
+	 *            - the index of the field in the list meta-data whose default value is to be converted.
+	 * @return The Java runtime value of the JCoList field's default value.
+	 */
+	public static Object convertJCoDefaultValue(JCoListMetaData listMetaData, int index) {
+		switch(listMetaData.getType(index)) {
+		case JCoMetaData.TYPE_INT:
+		case JCoMetaData.TYPE_INT1:
+		case JCoMetaData.TYPE_INT2:
+			return convertJcoIntegerDefaultValue(listMetaData.getDefault(index));
+
+		case JCoMetaData.TYPE_CHAR:
+		case JCoMetaData.TYPE_NUM:
+			return convertJCoStringDefaultValue(listMetaData.getDefault(index));
+
+		case JCoMetaData.TYPE_BCD:
```

Comment:
What do you think to merge with JCoMetaData.TYPE_DECF34 and JCoMetaData.TYPE_DECF16 ?